runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordAreaType.java

```diff
@@ -29,5 +29,7 @@
 	BOSSES,
 	CITIES,
 	DUNGEONS,
-	MINIGAMES;
+	MINIGAMES,
+	RAIDS,
```

Comment:
I dont like this special "types" of activities, maybe figure out a way how to not do this.